packages/sdks/overrides/rsc/src/helpers/preview-lru-cache/set.ts

```diff
@@ -1,19 +1,22 @@
 'use server';
 
+import { revalidatePath } from 'next/cache';
```

Comment:
@samijaber seems like the build is failing, did an initial research and found out that:
- we need to externalize `next/cache` so that it's not required to be bundled inside the build similar to `next/navigation`: https://github.com/BuilderIO/builder/blob/main/packages/sdks/output/nextjs/vite.config.ts#L45
- add a declaration module for types to export from next/cache.js for the `.js` implicit extension workaround: https://github.com/BuilderIO/builder/blob/main/packages/sdks/output/nextjs/module-augmentations.d.ts#L5

Reply:
Thanks for the help!